drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -323,6 +333,7 @@ local function do_configure(driver, device)
   local thermo_eps = device:get_endpoints(clusters.Thermostat.ID)
   local fan_eps = device:get_endpoints(clusters.FanControl.ID)
   local wind_eps = device:get_endpoints(clusters.FanControl.ID, {feature_bitmap = clusters.FanControl.types.FanControlFeature.WIND})
+  local rock_eps = device:get_endpoints(clusters.FanControl.ID, {feature_bitmap = clusters.FanControl.types.FanControlFeature.ROCK})
```

Comment:
I looked at FanControl/types/Feature.lua in the lua libs, and it looks the feature name should be ROCKING rather than ROCK

Reply:
huh. you're so right. Weirdly, it onboarded correctly on a real device as well as passing the unit tests. should probably be changed though. good catch!